packages/playground/remote/remote.html

```diff
@@ -63,24 +63,61 @@
 				const div = document.createElement('div');
 				div.className = 'error-message';
 				div.innerText = 'Ooops! WordPress Playground had a hiccup!';
-				if (
-					e?.message?.includes(
-						'The user denied permission to use Service Worker'
-					)
-				) {
-					div.innerText =
-						'It looks like you have disabled third-party cookies in your browser. This tends to ' +
-						'also disable the ServiceWorker API used by WordPress Playground. Please re-enable ' +
-						'third-party cookies and try again.';
-				}
-
 				document.body.append(div);
 
 				const button = document.createElement('button');
 				button.innerText = 'Try again';
 				button.onclick = () => {
 					window.location.reload();
 				};
+				if (
+					e?.message?.includes(
+						'The user denied permission to use Service Worker'
```

Comment:
This may not be super stable. Can you check for the `e.name` instead, or is this ambiguous?

Reply:
Oh great idea! `e.name` is `NotSupportedError` which seems to be also [used for payments and screen orientation](https://github.com/search?q=repo%3Achromium%2Fchromium+NotSupportedError&type=code&p=2), but I think that's unambiguous enough for now. I also noticed `e.code` is set to `9` which seems to come from [blink::mojom::ServiceWorkerErrorType::kDisabled](https://github.com/chromium/chromium/blob/23e2c4f0f11df9c207e5290e83c2cecf877f12ba/content/browser/service_worker/service_worker_container_host.cc#LL1716C9-L1716C56) which is weird because I'd expect that to be 4 based on [its position in ServiceWorkerErrorType](https://chromium.googlesource.com/chromium/src/third_party/+/refs/heads/main/blink/public/mojom/service_worker/service_worker_error_type.mojom#14). 9 seems to be kSecurity.

Reply:
> Oh great idea! `e.name` is `NotSupportedError` which seems to be also [used for payments and screen orientation](https://github.com/search?q=repo%3Achromium%2Fchromium+NotSupportedError&type=code&p=2), but I think that's unambiguous enough for now. I also noticed `e.code` is set to `9` which seems to come from [blink::mojom::ServiceWorkerErrorType::kDisabled](https://github.com/chromium/chromium/blob/23e2c4f0f11df9c207e5290e83c2cecf877f12ba/content/browser/service_worker/service_worker_container_host.cc#LL1716C9-L1716C56) which is weird because I'd expect that to be 4 based on [its position in ServiceWorkerErrorType](https://chromium.googlesource.com/chromium/src/third_party/+/refs/heads/main/blink/public/mojom/service_worker/service_worker_error_type.mojom#14). 9 seems to be kSecurity.

The error name should definitely be the same across browsers, as it's [specified](https://privacycg.github.io/storage-access/#dom-document-requeststorageaccess:~:text=If%20doc%E2%80%99s%20active%20sandboxing%20flag%20set%20has%20its%20sandbox%20storage%20access%20by%20user%20activation%20flag%20set%2C%20reject%20p%20with%20a%20%22NotAllowedError%22%20DOMException%20and%20return%20p.). My understanding is that the vendors have some leeway when it comes to making the error message understandable to "their" developers. Not sure about the error code. Just use the name I guess :-)

Reply:
> Just use the name I guess :-)

Yup, sounds perfect - thank you so much  🙌

Reply:
Adjusted in 8d5193432a7cce541a7dd5f1c45fd44e7781d735